src/libraries/System.IO.Compression/src/System.IO.Compression.csproj

```diff
@@ -25,8 +25,10 @@
     <Compile Include="System\IO\Compression\DeflateZLib\DeflateStream.cs" />
     <Compile Include="System\IO\Compression\DeflateZLib\Inflater.cs" />
     <Compile Include="System\IO\Compression\DeflateZLib\ZLibException.cs" />
-    <Compile Include="System\IO\Compression\DeflateZLib\ZLibNative.cs" />
-    <Compile Include="System\IO\Compression\DeflateZLib\ZLibNative.ZStream.cs" />
+    <Compile Include="$(CommonPath)System\IO\Compression\ZLibNative.cs"
+             Link="Common\System\IO\Compression\ZLibNative.cs" />
+    <Compile Include="$(CommonPath)System\IO\Compression\ZLibNative.ZStream.cs"
+             Link="Common\System\IO\Compression\ZLibNative.ZStream.cs" />
```

Comment:
Thanks you @CarnaViire for the early heads up to the Compression area owners that you were intending to move these files.